Makefile

```diff
@@ -749,18 +750,18 @@ doc : doc-html
 
 # Style/Format
 CLANG_FORMAT ?= clang-format
-
-FORMAT_OPTS += -style=file -i
+CLANG_FORMAT_OPTS += -style=file -i
+AUTOPEP8 ?= autopep8
+AUTOPEP8_OPTS += --in-place --aggressive
```

Comment:
If I recall correctly, for `tests/junit.py` we also bumped the line length limit to 120

Reply:
Do you want me to add that here then? As in `AUTOPEP8_OPTS += --in-place --aggressive --max-line-length 120`. Or am I misunderstading?

Reply:
I think it's worth seeing how it looks. It probably won't change much and then we could grep for all .py files in the repo and apply the same formatting to everything

Reply:
Confirmed it changes nothing :)

Reply:
Here's the diff:

```
diff --git a/Makefile b/Makefile
index d6ae2d1a..0f996324 100644
--- a/Makefile
+++ b/Makefile
@@ -746,10 +746,10 @@ doc : doc-html
 CLANG_FORMAT ?= clang-format
 CLANG_FORMAT_OPTS += -style=file -i
 AUTOPEP8 ?= autopep8
-AUTOPEP8_OPTS += --in-place --aggressive
+AUTOPEP8_OPTS += --in-place --aggressive --max-line-length 120

 format.ch := $(filter-out include/ceedf.h $(wildcard tests/t*-f.h), $(shell git ls-files '*.[ch]pp' '*.[ch]'))
-format.py := $(filter-out tests/junit-xml/junit_xml/__init__.py $(wildcard tests/*.py), $(shell git ls-files '*.py'))
+format.py := $(filter-out tests/junit-xml/junit_xml/__init__.py, $(shell git ls-files '*.py'))

 format-c  :
        $(CLANG_FORMAT) $(CLANG_FORMAT_OPTS) $(format.ch)
diff --git a/tests/junit.py b/tests/junit.py
index 1f050bff..9d2ef983 100755
--- a/tests/junit.py
+++ b/tests/junit.py
@@ -9,8 +9,20 @@ def create_argparser() -> argparse.ArgumentParser:
         argparse.ArgumentParser: Created `ArgumentParser`
     """
     parser = argparse.ArgumentParser('Test runner with JUnit and TAP output')
-    parser.add_argument('-c', '--ceed-backends', type=str, nargs='*', default=['/cpu/self'], help='libCEED backend to use with convergence tests')
-    parser.add_argument('-m', '--mode', type=RunMode, action=CaseInsensitiveEnumAction, help='Output mode, junit or tap', default=RunMode.JUNIT)
+    parser.add_argument(
+        '-c',
+        '--ceed-backends',
+        type=str,
+        nargs='*',
+        default=['/cpu/self'],
+        help='libCEED backend to use with convergence tests')
+    parser.add_argument(
+        '-m',
+        '--mode',
+        type=RunMode,
+        action=CaseInsensitiveEnumAction,
+        help='Output mode, junit or tap',
+        default=RunMode.JUNIT)
     parser.add_argument('-n', '--nproc', type=int, default=1, help='number of MPI processes')
     parser.add_argument('-o', '--output', type=Optional[Path], default=None, help='Output file to write test')
     parser.add_argument('-b', '--junit-batch', type=str, default='', help='Name of JUnit batch for output file')
@@ -84,7 +96,8 @@ class CeedSuiteSpec(SuiteSpec):
         Returns:
             Optional[str]: Skip reason, or `None` if test case should not be skipped
         """
-        if contains_any(resource, ['occa']) and startswith_any(test, ['t4', 't5', 'ex', 'mfem', 'nek', 'petsc', 'fluids', 'solids']):
+        if contains_any(resource, ['occa']) and startswith_any(
+                test, ['t4', 't5', 'ex', 'mfem', 'nek', 'petsc', 'fluids', 'solids']):
             return 'OCCA mode not supported'
         if test.startswith('t318') and contains_any(resource, ['/gpu/cuda/ref']):
             return 'CUDA ref backend not supported'
diff --git a/tests/junit_common.py b/tests/junit_common.py
index af1ed1fa..5043eac6 100644
--- a/tests/junit_common.py
+++ b/tests/junit_common.py
@@ -388,7 +388,8 @@ def run_tests(test: str, ceed_backends: list[str], mode: RunMode, nproc: int, su

             # check required failures
             if not test_case.is_skipped():
-                required_message, did_fail = suite_spec.check_required_failure(test, spec, ceed_resource, test_case.stderr)
+                required_message, did_fail = suite_spec.check_required_failure(
+                    test, spec, ceed_resource, test_case.stderr)
                 if required_message and did_fail:
                     test_case.status = f'fails with required: {required_message}'
                 elif required_message:
```

Reply:
Looks good to me